src/main/java/net/eldeen/dropwizard/CfSignalResourceBundle.java

```diff
@@ -34,9 +26,14 @@
    * @param awsRegion       the aws region the stack is in, e.g. us-west-2
    */
   public CfSignalResourceBundle(String asgResourceName, String stackName, String awsRegion) {
+    this(asgResourceName, new CloudFormationFacade(stackName, awsRegion));
```

Comment:
This does setup nicely for making this bundle do configuration the dropwizard way. :+1: